index.html

```diff
@@ -0,0 +1,751 @@
+                        general. The study of nature is a large, if not the only, part of science. Although humans are
+                        part
+                        of
+                        nature, human activity is often understood as a separate category from other natural phenomena.
+                    </p>
+                    <button type="button" class="btn btn-secondary">Read More</button>
+                    <button type="button" class="btn btn-secondary btn-dismiss">x</button>
+                    <small class="txt-overlay">Bachalpsee in the Swiss Alps</small>
+                </div>
+                <div class="code-frame"><iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28219%2C234%2C254%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2522%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522card-img%2522%2520alt%253D%2522card-image%2522%250A%2520%2520%2520%2520%2520%2520%2520%2520src%253D%2522img-location.jpg%2522%253E%250A%2520%2520%2520%2520%253Ch3%2520class%253D%2522card-heading%2522%253ENature%2520Photography%253C%252Fh3%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522card-desc%2522%253E%250A%2520%2520%2520%2520%2520%2520%2522Nature%2522%2520can%2520refer%2520to%2520the%2520phenomena%2520of%2520the%2520physical%2520world%252C%250A%2520%2520%2520%2520%2520%2520and%2520also%2520to%2520life%2520in%2520general.%2520The%2520study%2520of%2520nature%2520is%2520a%2520large%252C%250A%2520%2520%2520%2520%2520%2520if%2520not%2520the%2520only%252C%2520part%2520of%2520science.%2520Although%2520humans%2520are%2520part%250A%2520%2520%2520%2520%2520%2520%2520%2520of%2520nature%252C%2520human%2520activity%2520is%2520often%2520understood%2520as%2520a%2520separate%250A%2520%2520%2520%2520%2520%2520category%2520from%2520other%2520natural%2520phenomena.%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cbutton%2520type%253D%2522button%2522%2520class%253D%2522btn%2520btn-secondary%2522%253E%250A%2520%2520%2520%2520%2520%2520Read%2520More%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Cbutton%2520type%253D%2522button%2522%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520class%253D%2522btn%2520btn-secondary%2520btn-dismiss%2522%253Ex%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Csmall%2520class%253D%2522txt-overlay%2522%253E%250A%2520%2520%2520%2520%2520%2520Bachalpsee%2520in%2520the%2520Swiss%2520Alps%253C%252Fsmall%253E%250A%253C%252Fdiv%253E"
+                        style="width: 715px; height: 499px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe></div>
+                <h3>Horizontal Card</h3>
+                <div class="card-horizontal">
```

Comment:
For horizontal card, Flex would do good, I dont feel any need for using grid here

Reply:
Here, we would have a flex inside flex right?
One flex to keep the image and text container next to each other.
Second flex to align the contents within the text-container. To avoid this, I used a single grid.

Reply:
Yes, would require 2 flex here.